src/mmrelay/cli.py

```
@@ -1262,8 +1290,34 @@ def handle_auth_login(args):
+            encryption_config = matrix_section.get("encryption", {})
+            e2ee_enabled = e2ee_config.get("enabled", False) or encryption_config.get(
+                "enabled", False
+            )
+
+            if e2ee_enabled:
+                print("Matrix Bot Authentication for E2EE")
+                print("===================================")
+            else:
+                print("Matrix Bot Authentication")
+                print("=========================")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)